### ISSUE TITLE
Add support for Avro IDL

### ISSUE THREAD

**Issue by mikaello** [mikaello/avrodoc-plus#4]
Avro schemas can be defined in both JSON and [Avro IDL](https://avro.apache.org/docs/current/idl.html), avrodoc-plus only supports JSON, but it would be nice if we could support Avro IDL as well.

**Comment by karstenspang** [comment]
Agree, that would be nice. Currently, I generate JSON files just for the purpose of Avrodoc. This works fine, except that the file name column on the overview page is useless, as there are no files in the source tree with those names.
